packages/itwinui-css/src/color-picker/color-picker.scss

```diff
@@ -4,18 +4,22 @@
 @use '../button/borderless';
 @use '../mixins';
 
-@mixin iui-transparent-background($size: var(--iui-size-m)) {
-  background-position: 0 0, calc($size * 0.5) calc($size * 0.5);
-  background-size: $size $size;
-  background-image: repeating-linear-gradient(
-    45deg,
-    #C7CCD1 25%,
-    transparent 25%,
-    transparent 75%,
-    #C7CCD1 75%,
-    #C7CCD1
-  ),
-    repeating-linear-gradient(45deg, #C7CCD1 25%, #EDEFF2 25%, #EDEFF2 75%, #C7CCD1 75%, #C7CCD1);
+@mixin iui-transparent-background($size: var(--iui-size-m), $variant: swatch) {
```

Comment:
@mayank99, this was my attempt at getting [your opacity slider gradient suggestion](https://github.com/iTwin/iTwinUI/pull/1469#discussion_r1332203652) to work with this mixin.  This did also remove a `::after` from the swatch.  I would not be surprised if there is a cleaner way to achieve this, but this is what I was able to cook up.

Reply:
why complicate this so much? it's not worth it. just duplicate the code. it's getting duplicated in the final css anyway

Reply:
I've removed the mixin now.